### PR TITLE
Merges master in v3 branch

### PR DESCRIPTION
**Pull request by taylorjdawson**
Steps taken:
 - locally merged master into v3
 - created a fork of go-capnproto2 and pushed local v3 changes to my fork's v3 branch
 - opened this PR so that we can all see the results of my merge from master to v3 before committing it

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# How to contribute
+
+As of 2020-09-13, Ross Light, the primary maintainer, does not have enough time
```
Comment (lthibault) [inline CONTRIBUTING.md:3]: I think this paragraph can be removed.  Perhaps we should add an official list of maintainers at the top of this file, namely:  myself, Ian and Taylor ?

File: pointer.go [hunk 2]
```diff
@@ -144,7 +144,7 @@ func (p Ptr) text() (b []byte, ok bool) {
 		// Text must be null-terminated.
 		return nil, false
 	}
-	return b[:len(b)-1 : len(b)], true
+	return b[: len(b)-1 : len(b)], true
```
Comment (lthibault) [inline pointer.go:147]: `gofmt` ?

**Review by lthibault — COMMENTED** [review]
This **looks** good to me, but I leave it to @zenhack to sanity-check and approve.

My comments are minor and can be addressed in future PRs.  Please don't block over them!

@taylorjdawson Just a friendly reminder to double-check that the tests pass before merging, since CI is temporarily disabled.

**Comment by zenhack** [comment]
I see some build failures:

```
$ go test ./...
# capnproto.org/go/capnp/v3
./segment.go:203:19: undefined: errListSize
./segment.go:221:18: undefined: errListSize
# capnproto.org/go/capnp/v3 [capnproto.org/go/capnp/v3.test]
./segment.go:203:19: undefined: errListSize
./segment.go:221:18: undefined: errListSize
FAIL	capnproto.org/go/capnp/v3 [build failed]
FAIL	capnproto.org/go/capnp/v3/capnpc-go [build failed]
FAIL	capnproto.org/go/capnp/v3/encoding/text [build failed]
FAIL	capnproto.org/go/capnp/v3/internal/demo [build failed]
ok  	capnproto.org/go/capnp/v3/internal/errors	(cached)
ok  	capnproto.org/go/capnp/v3/internal/packed	2.176s
FAIL	capnproto.org/go/capnp/v3/pogs [build failed]
FAIL	capnproto.org/go/capnp/v3/rpc [build failed]
FAIL	capnproto.org/go/capnp/v3/schemas [build failed]
FAIL	capnproto.org/go/capnp/v3/server [build failed]
```

**Comment by taylorjdawson** [comment]
@zenhack yea I am resolving those now 👍🏿

**Comment by lthibault** [comment]
Closing in light of #172.

**Pull request closed without merging**